### ISSUE TITLE
Text request

### ISSUE THREAD

**Issue by schmidt-anja** [tls-kr/tls-app#215]
Hello, 

I have recently thought about adding existing translations of the 華嚴經問答 (T45n1873) (KR6e0087) to the database, but I have found that the text is no longer available. Could the text be added back, or, alternatively, will the option of adding translations and annotations to Kanripo texts in the TLS become available soon? 

Thank you!

**Comment by schmidt-anja** [comment]
Thank you very much! I am getting the error code 1 (dispseg) when I try to go farther than page 2, which I can see is a known issue from the pinned post. 
Also, while translating so far I have noticed a few mistakes with the punctuation, would it be possible for me to get text editing permissions for just this text?

**Comment by cwittern** [comment]
OK, I have given you editing permission.  I hope I can make a documentation page for this in the next week. 

As for the dispseg error code, that is a different issue.  I do not know exactly why it appears, but for this specific case, it seems you can work around this as follows (ugly, I know, but it seems to work for now):   
In the address bar of your browser, you should see a line like this:
https://hxwd.org/textview.html?location=T45n1873_CBETA_001-0598c1501.s17&prec=2&foll=28
if the problem occurs, removing the part starting from the first '&' to the end like this:
https://hxwd.org/textview.html?location=T45n1873_CBETA_001-0598c1501.s17
apparently allows the page to display.  

I will have to investigate the problem, but at least you should be able to move through the text like this.

**Comment by schmidt-anja** [comment]
Thank you for your help, that works.

**Comment by cwittern** [comment]
Actually, I think I found the problem, let's see if it is fixed now.

**Comment by schmidt-anja** [comment]
Looks good!

**Comment by matchachoco95** [comment]
Another text seems to have disappeared as well: I'm quite sure the 冥報記 (T51n2082_p0787c01) used to be available in TLS but now it only shows two references to Kanripo. Could that one be restored?

**Comment by matchachoco95** [comment]
Oh, and Christoph asked me to ask you to add the following text so it could be annotated: KR3a0069 孔子集語 

(Perhaps a permanent "Text requests" thread would make sense? Then again, I know this is usually supposed to be done via the "request" button in the database...)

**Comment by cwittern** [comment]
> Another text seems to have disappeared as well: I'm quite sure the 冥報記 (T51n2082_p0787c01) used to be available in TLS but now it only shows two references to Kanripo. Could that one be restored?
Done

**Comment by cwittern** [comment]
> Oh, and Christoph asked me to ask you to add the following text so it could be annotated: KR3a0069 孔子集語

Yeah, I will look into this.  The text request system is currently on vacation.
